### PR TITLE
chore: Adding webUri input parameter to ConfigWatcher function

### PR DESCRIPTION
**Pull request by gatici**
- Adding webUri input parameter to ConfigWatcher function to parameterise Webui URL.
- Adding golangci.yml for linting checks 
-  Adding GHA for linting

### CODE REVIEW

File: Makefile
Comment (gab-arrobo) [inline Makefile:1]: Why do you need to add this file?

File: .golangci.yml
Comment (gab-arrobo) [inline .golangci.yml:1]: Given that you are adding this file, what about adding a `lint` action for GHA (similar to this: https://github.com/omec-project/amf/blob/master/.github/workflows/master.yml#L35)?

File: .golangci.yml [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+  # autogenerated files. If it's not please let us know.
+  # "/" will be replaced by current OS file path separator to properly work
+  # on Windows.
+  skip-files:
+    - "api_.*\\.go$"
+    - "model_.*\\.go$"
+    - "routers.go"
+    - "client.go"
+    - "configuration.go"
+    - "nas.go"
```
Comment (gab-arrobo) [inline .golangci.yml:43]: I think these are not applicable, correct?

File: .golangci.yml [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+    - whitespace
+    # - unconvert
+    - predeclared
+    # - noctx
+    - dogsled
+    # - bodyclose
+    - asciicheck
+      #- stylecheck
+      # - unparam
+    #- wsl
```
Comment (gab-arrobo) [inline .golangci.yml:247]: Are you planning to open other PRs to enable additional linters?

File: Makefile
Comment (gatici) [reply Makefile:1]: We can remove it.

Comment (gatici) [reply Makefile:1]: Done

File: .golangci.yml
Comment (gatici) [reply .golangci.yml:1]: Added.

File: .golangci.yml (hunk 2 above)
Comment (gatici) [reply .golangci.yml]: Yes, we need to activate disabled linters one by one.

File: .golangci.yml
Comment (gab-arrobo) [reply .golangci.yml:1]: Thanks!

File: .golangci.yml (hunk 1 above)
Comment (gatici) [reply .golangci.yml]: Removed them.

File: go/config.pb.go [hunk 3]
```diff
@@ -2,7 +2,6 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-
```
Comment (gab-arrobo) [inline go/config.pb.go:5]: I think this needs to be reverted? The next line indicates that this file was generated by `protoc-gen-go`. So, I think we should need to skip the `*.pb.go` files

File: .golangci.yml (hunk 1 above)
Comment (gab-arrobo) [reply .golangci.yml]: Thanks. but I think you also removed the `*.pb.go`, which I think we should keep due to some files are automatically generated by `protoc-gen-go`

File: .golangci.yml (hunk 2 above)
Comment (gab-arrobo) [reply .golangci.yml]: Ok, thanks!

File: go/config.pb.go (hunk 3 above)
Comment (gatici) [reply go/config.pb.go]: The change is reverted and files are skipped from linting check.

**Review by gab-arrobo — APPROVED** [review]
+1 (tested changes with OnRamp using `replace` directive for the `ausf` image

**Pull request merged**